Infrastructure/gpg-ipdeny/README.md

```diff
@@ -20,25 +20,16 @@ Before running the deployment script you must be logged into GOV.UK PaaS using t
 The deployment script takes the following parameters:
 * -a PROTECTED_APP_NAME The application to protect
 * -e PROTECTED_APP_SPACE_NAME The name of the space the apps are in
-* -i DENIED_IPS A comma separated list of IP addresses to be banned
+* -f DENIED_IPS_FILENAME The filename containing the list of IPs to block
 * -r ROUTE_SERVICE_APP_NAME The name of the service app to create/update
 * -s ROUTE_SERVICE_NAME The name of the service to create/update
 
-For example to set up ip denylisting on the dev space the command could be run (using the gitbash window set up with jq earlier) as 
+For example to set up ip denylisting on the dev space, if you've created/placed the denylist file in the same directory as the deployment script, the command could be run (using the gitbash window set up with jq earlier) as 
 ```
-./deploy.sh -e gpg-dev -r gpg-dev-ipdeny -s gpg-dev-ipdeny-service -a gender-pay-gap-dev -i "1.2.3.4,2.3.4.5"
+./deploy.sh -e gpg-dev -r gpg-dev-ipdeny -s gpg-dev-ipdeny-service -a gender-pay-gap-dev -f "GPG IP Denylist.txt"
```

Comment:
I'm not a big fan of spaces in filenames but not massively fussed about changing it